test/StockportContentApiTests/Unit/Repositories/EventRepositoryTest.cs

```diff
@@ -698,5 +698,35 @@ public async void ShouldReturnEventsByTagIfNoCategoryPresent()
         var eventResponse = result.Get<List<Event>>();
         eventResponse.Count.Should().Be(3);
     }
+
+    [Fact]
+    public void EventsShourReturnInOrderOfStartDate()
```

Comment:
Typo in the Test name, and could we follow the usual structure of MethodName_ShouldDoThisThing?